packages/opentelemetry-types/src/metrics/Metric.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Resource } from '../resources/Resource';
+
+/**
+ * Options needed for metric creation
+ */
+export interface MetricOptions {
+  /** The description of the Metric. */
+  description?: string;
```

Comment:
Should this be required?

Reply:
I don't believe so since this is different from `name`

Reply:
As per the spec `description` is an optional param and the default to "".